Classes/Neos/Neos/Ui/Domain/Model/Feedback/Operations/ReloadContentOutOfBand.php

```diff
@@ -0,0 +1,146 @@
+<?php
+namespace Neos\Neos\Ui\Domain\Model\Feedback\Operations;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Neos\Ui\Domain\Model\FeedbackInterface;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+use Neos\Neos\View\FusionView as FusionView;
+use Neos\Flow\Mvc\Controller\ControllerContext;
+use Neos\Neos\Ui\Domain\Model\RenderedNodeDomAddress;
+use Neos\Fusion\Core\Cache\ContentCache;
+
+class ReloadContentOutOfBand implements FeedbackInterface
+{
+    /**
+     * @var NodeInterface
+     */
+    protected $node;
+
+    /**
+     * The node dom address
+     *
+     * @var RenderedNodeDomAddress
+     */
+    protected $nodeDomAddress;
+
+    /**
+     * @Flow\Inject
+     * @var ContentCache
+     */
+    protected $contentCache;
+
+    /**
+     * Set the node
+     *
+     * @param NodeInterface $node
+     * @return void
+     */
+    public function setNode(NodeInterface $node)
+    {
+        $this->node = $node;
+    }
+
+    /**
+     * Get the node
+     *
+     * @return NodeInterface
+     */
+    public function getNode()
+    {
+        return $this->node;
+    }
+
+    /**
+     * Set the node dom address
+     *
+     * @param RenderedNodeDomAddress $nodeDomAddress
+     * @return void
+     */
+    public function setNodeDomAddress(RenderedNodeDomAddress $nodeDomAddress = null)
+    {
+        $this->nodeDomAddress = $nodeDomAddress;
+    }
+
+    /**
+     * Get the node dom address
+     *
+     * @return RenderedNodeDomAddress
+     */
+    public function getNodeDomAddress()
+    {
+        return $this->nodeDomAddress;
+    }
+
+    /**
+     * Get the type identifier
+     *
+     * @return string
+     */
+    public function getType()
+    {
+        return 'Neos.Neos.Ui:ReloadContentOutOfBand';
+    }
+
+    /**
+     * Get the description
+     *
+     * @return string
+     */
+    public function getDescription()
+    {
+        return sprintf('Rendering of node "%s" required.', $this->getNode()->getPath());
+    }
+
+    /**
+     * Checks whether this feedback is similar to another
+     *
+     * @param FeedbackInterface $feedback
+     * @return boolean
+     */
+    public function isSimilarTo(FeedbackInterface $feedback)
+    {
+        if (!$feedback instanceof ReloadContentOutOfBand) {
+            return false;
+        }
+
+        return (
+            $this->getNode()->getContextPath() === $feedback->getNode()->getContextPath() &&
+            $this->getReferenceData() == $feedback->getReferenceData()
+        );
+    }
+
+    /**
+     * Serialize the payload for this feedback
+     *
+     * @return mixed
+     */
+    public function serializePayload(ControllerContext $controllerContext)
+    {
+        return [
+            'contextPath' => $this->getNode()->getContextPath(),
+            'nodeDomAddress' => $this->getNodeDomAddress(),
+            'renderedContent' => $this->renderContent($controllerContext)
```

Comment:
This is resulting in an error *sometimes*
When I try to edit the id of a youtube video it is working, but when I try to edit the title of the header image of the demo page I get this error.  I'm on dev-distribution current master(updated today) - I'm not sure if the error has more to do with Neos.Fusion or the UI.

Stacktrace:

```

Exception #1332493990 in line 393 of /home/maex/projects/contrib/neos-development-distribution/Data/Temporary/Development/Cache/Code/Flow_Object_Classes/Neos_Fusion_Core_Runtime.php: No Fusion object found in path ""
					Please make sure to define one in your Fusion configuration.

35 Neos\Fusion\Core\Runtime_Original::throwExceptionForUnrenderablePathIfNeeded("", array|0|, "Exception")
34 Neos\Fusion\Core\Runtime_Original::evaluateInternal("", "Exception")
33 Neos\Fusion\Core\Runtime_Original::render("")
32 Neos\Neos\View\FusionView_Original::render()
31 Neos\Neos\Ui\Domain\Model\Feedback\Operations\ReloadContentOutOfBand_Original::renderContent(Neos\Flow\Mvc\Controller\ControllerContext)
30 Neos\Neos\Ui\Domain\Model\Feedback\Operations\ReloadContentOutOfBand_Original::serializePayload(Neos\Flow\Mvc\Controller\ControllerContext)
29 Neos\Neos\Ui\Domain\Model\FeedbackCollection_Original::jsonSerialize()
28 Neos\Flow\Mvc\View\JsonView_Original::transformValue(Neos\Neos\Ui\Domain\Model\FeedbackCollection, array|0|)
27 Neos\Flow\Mvc\View\JsonView_Original::renderArray()
26 Neos\Flow\Mvc\View\JsonView_Original::render()
25 Neos\Flow\Mvc\Controller\ActionController_Original::callActionMethod()
24 Neos\Neos\Ui\Controller\BackendServiceController::callActionMethod()
23 call_user_func_array(array|2|, array|0|)
22 Neos\Neos\Ui\Controller\BackendServiceController::Flow_Aop_Proxy_invokeJoinPoint(Neos\Flow\Aop\JoinPoint)
21 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
20 Neos\Flow\Security\Aspect\PolicyEnforcementAspect_Original::enforcePolicy(Neos\Flow\Aop\JoinPoint)
19 Neos\Flow\Aop\Advice\AroundAdvice::invoke(Neos\Flow\Aop\JoinPoint)
18 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
17 Neos\Neos\Ui\Controller\BackendServiceController::callActionMethod()
16 Neos\Flow\Mvc\Controller\ActionController_Original::processRequest(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
15 Neos\Neos\Ui\Controller\BackendServiceController::processRequest(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
14 call_user_func_array(array|2|, array|2|)
13 Neos\Neos\Ui\Controller\BackendServiceController::Flow_Aop_Proxy_invokeJoinPoint(Neos\Flow\Aop\JoinPoint)
12 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
11 Neos\Flow\Security\Aspect\PolicyEnforcementAspect_Original::enforcePolicy(Neos\Flow\Aop\JoinPoint)
10 Neos\Flow\Aop\Advice\AroundAdvice::invoke(Neos\Flow\Aop\JoinPoint)
9 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
8 Neos\Neos\Ui\Controller\BackendServiceController::processRequest(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
7 Neos\Flow\Mvc\Dispatcher_Original::initiateDispatchLoop(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
6 Neos\Flow\Mvc\Dispatcher_Original::dispatch(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
5 Neos\Flow\Mvc\DispatchComponent_Original::handle(Neos\Flow\Http\Component\ComponentContext)
4 Neos\Flow\Http\Component\ComponentChain_Original::handle(Neos\Flow\Http\Component\ComponentContext)
3 Neos\Flow\Http\Component\ComponentChain_Original::handle(Neos\Flow\Http\Component\ComponentContext)
2 Neos\Flow\Http\RequestHandler::handleRequest()
1 Neos\Flow\Core\Bootstrap::run()


HTTP REQUEST:
POST /neos/ui-services/change HTTP/1.1
Host: 127.0.0.1:8000
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Referer: http://127.0.0.1:8000/neos/content?node=/sites/neosdemo/the-book/vi-pig-and-pepper@user-admin;country=eng&language=en_US
X-Flow-Csrftoken: 30b9f4560b306c77e8fb80b5e4a51060
Content-Type: application/json
Origin: http://127.0.0.1:8000
Content-Length: 926
Connection: keep-alive

{"changes":[{"type":"Neos.Neos.Ui:Property","subject":"/sites/neosdemo/the-book/vi-pig-and-pepper@user-admin;country=eng&language=en_US","payload":{"propertyName":"chapterDescription","value":"aaaaFor a minute or two she stood looking at the house,\n                                    and wondering what to do\n       aaaaa                             next, when suddenly a footman in livery came running out of the wood—(she considered\n                                    him to be a\n                                    footman because he was in livery: otherwise, judging by his face only, she would\n                                    have called him\n                                    a fish)—and rapped loudly at the door with his knuckles.\n                                ","nodeDomAddress":{"contextPath":"/sites/neosdemo/the-book/vi-pig-and-pepper@user-admin;country=eng&language=en_US","fusionPath":""}}}]}

HTTP RESPONSE:
[response was empty]

PHP PROCESS:
Inode: 4463318
PID: 28253
UID: 1000
GID: 990
User: maex

    [array] => 

Exception #1332493990 in line 393 of /home/maex/projects/contrib/neos-development-distribution/Data/Temporary/Development/Cache/Code/Flow_Object_Classes/Neos_Fusion_Core_Runtime.php: No Fusion object found in path ""
					Please make sure to define one in your Fusion configuration.

35 Neos\Fusion\Core\Runtime_Original::throwExceptionForUnrenderablePathIfNeeded("", array|0|, "Exception")
34 Neos\Fusion\Core\Runtime_Original::evaluateInternal("", "Exception")
33 Neos\Fusion\Core\Runtime_Original::render("")
32 Neos\Neos\View\FusionView_Original::render()
31 Neos\Neos\Ui\Domain\Model\Feedback\Operations\ReloadContentOutOfBand_Original::renderContent(Neos\Flow\Mvc\Controller\ControllerContext)
30 Neos\Neos\Ui\Domain\Model\Feedback\Operations\ReloadContentOutOfBand_Original::serializePayload(Neos\Flow\Mvc\Controller\ControllerContext)
29 Neos\Neos\Ui\Domain\Model\FeedbackCollection_Original::jsonSerialize()
28 Neos\Flow\Mvc\View\JsonView_Original::transformValue(Neos\Neos\Ui\Domain\Model\FeedbackCollection, array|0|)
27 Neos\Flow\Mvc\View\JsonView_Original::renderArray()
26 Neos\Flow\Mvc\View\JsonView_Original::render()
25 Neos\Flow\Mvc\Controller\ActionController_Original::callActionMethod()
24 Neos\Neos\Ui\Controller\BackendServiceController::callActionMethod()
23 call_user_func_array(array|2|, array|0|)
22 Neos\Neos\Ui\Controller\BackendServiceController::Flow_Aop_Proxy_invokeJoinPoint(Neos\Flow\Aop\JoinPoint)
21 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
20 Neos\Flow\Security\Aspect\PolicyEnforcementAspect_Original::enforcePolicy(Neos\Flow\Aop\JoinPoint)
19 Neos\Flow\Aop\Advice\AroundAdvice::invoke(Neos\Flow\Aop\JoinPoint)
18 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
17 Neos\Neos\Ui\Controller\BackendServiceController::callActionMethod()
16 Neos\Flow\Mvc\Controller\ActionController_Original::processRequest(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
15 Neos\Neos\Ui\Controller\BackendServiceController::processRequest(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
14 call_user_func_array(array|2|, array|2|)
13 Neos\Neos\Ui\Controller\BackendServiceController::Flow_Aop_Proxy_invokeJoinPoint(Neos\Flow\Aop\JoinPoint)
12 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
11 Neos\Flow\Security\Aspect\PolicyEnforcementAspect_Original::enforcePolicy(Neos\Flow\Aop\JoinPoint)
10 Neos\Flow\Aop\Advice\AroundAdvice::invoke(Neos\Flow\Aop\JoinPoint)
9 Neos\Flow\Aop\Advice\AdviceChain::proceed(Neos\Flow\Aop\JoinPoint)
8 Neos\Neos\Ui\Controller\BackendServiceController::processRequest(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
7 Neos\Flow\Mvc\Dispatcher_Original::initiateDispatchLoop(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
6 Neos\Flow\Mvc\Dispatcher_Original::dispatch(Neos\Flow\Mvc\ActionRequest, Neos\Flow\Http\Response)
5 Neos\Flow\Mvc\DispatchComponent_Original::handle(Neos\Flow\Http\Component\ComponentContext)
4 Neos\Flow\Http\Component\ComponentChain_Original::handle(Neos\Flow\Http\Component\ComponentContext)
3 Neos\Flow\Http\Component\ComponentChain_Original::handle(Neos\Flow\Http\Component\ComponentContext)
2 Neos\Flow\Http\RequestHandler::handleRequest()
1 Neos\Flow\Core\Bootstrap::run()


HTTP REQUEST:
POST /neos/ui-services/change HTTP/1.1
Host: 127.0.0.1:8000
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Referer: http://127.0.0.1:8000/neos/content?node=/sites/neosdemo/the-book/vi-pig-and-pepper@user-admin;country=eng&language=en_US
X-Flow-Csrftoken: 30b9f4560b306c77e8fb80b5e4a51060
Content-Type: application/json
Origin: http://127.0.0.1:8000
Content-Length: 926
Connection: keep-alive

{"changes":[{"type":"Neos.Neos.Ui:Property","subject":"/sites/neosdemo/the-book/vi-pig-and-pepper@user-admin;country=eng&language=en_US","payload":{"propertyName":"chapterDescription","value":"aaaaFor a minute or two she stood looking at the house,\n                                    and wondering what to do\n       aaaaa                             next, when suddenly a footman in livery came running out of the wood—(she considered\n                                    him to be a\n                                    footman because he was in livery: otherwise, judging by his face only, she would\n                                    have called him\n                                    a fish)—and rapped loudly at the door with his knuckles.\n                                ","nodeDomAddress":{"contextPath":"/sites/neosdemo/the-book/vi-pig-and-pepper@user-admin;country=eng&language=en_US","fusionPath":""}}}]}

HTTP RESPONSE:
[response was empty]

PHP PROCESS:
Inode: 4463318
PID: 28253
UID: 1000
GID: 990
User: maex

    [array] => 

```

Reply:
Thanks, will try to reproduce this evening!